src/PrestaShopBundle/Translation/View/TranslationsTreeBuilder.php

```diff
@@ -0,0 +1,172 @@
+<?php
+/**
+ * 2007-2020 PrestaShop SA and Contributors
```

Comment:
I think the headers are outdated, we now have a "without ending date" version, see https://github.com/PrestaShop/PrestaShop/blob/develop/src/PrestaShopBundle/Command/UpdateLicensesCommand.php#L41

You can use `prestashop:licenses:update` to fix all files and then configure your IDE to apply the right header 😉 

What's more, the year is 2021 😛